### ISSUE TITLE
Loop Issue

### ISSUE THREAD

**Issue by bzmillerboy** [hutsoninc/gatsby-plugin-csv-feed#1]
Thanks for the Gatsby plugin! I'm however getting output that is creating duplicate columns in addition to a row for each record. Essentially, my output has a column record for each number of total records. Any thoughts on why this might be?

Here is my code:
```
{
      resolve: 'gatsby-plugin-csv-feed',
      options: {
        // Feeds
        feeds: [
          {
            // Individual feed query
            query: `
              {
                allSanityInventory(
                  filter: {
                    slug: { current: { ne: null } }
                    equipmentCategories: { slug: { current: { ne: null } } }
                  }
                ) {
                  edges {
                    node {
                      id
                      title
                      stockNumber
                      slug {
                        current
                      }
                      equipmentCategories {
                        slug {
                          current
                        }
                      }
                    }
                  }
                }
              }
            `,
            serialize: ({ query: { allSanityInventory } }) => {
              return allSanityInventory.edges.map((edge) => {
                const node = edge.node
                console.log('csv node:', node)
                return {
                  'Number ID': node.id,
                  'Item title': node.title,
                  'Item Stock Number': node.stockNumber,
                }
              })
            },
            // Output file
            output: '/inventory-feed.csv',
            // Options to pass to `json2csv` parser for this feed (optional)
            parserOptions: {},
          },
        ],
      },
    },
```
![Screenshot](https://pouch.jumpshare.com/preview/LKCRwCkCEFyT9L657EXyMlrFLilgcJZRcaBYDn0fUW6DC45lZAc2a5kFbC4Q-jIJksowy8iNdvWEKiP1yZNCdDJprjKJn4snkOQIlHyX-Do)

**Comment by AustinLeeGordon** [comment]
Thanks for filing an issue @bzmillerboy! Looks like the problem is with line 46 below.

https://github.com/hutsoninc/gatsby-plugin-csv-feed/blob/7fea5440ca99df2f1104bb4e40cd3ed4c21e6ddb/src/gatsby-node.js#L43-L51

I'll push a fix and let you know when the new version is released!

**Comment by AustinLeeGordon** [comment]
@bzmillerboy Should be fixed in [`v4.0.1`](https://www.npmjs.com/package/gatsby-plugin-csv-feed/v/4.0.1). Let me know if you have any more issues!

**Comment by bzmillerboy** [comment]
Yes, that indeed fixed the issue. Thanks for the quick turnaround and response!
